app/assets/javascripts/petition-widget-builder.js

```diff
@@ -0,0 +1,103 @@
+(function($) {
+
+  // This is not good, but I would not like to make a request to fetch this HTML
+  var template = [
+    "<div class='petition-widget-builder'>",
+    "  <h2> Incorporar petição </h2>",
```

Comment:
yes, I would have transformed this in uppercase on `css`, but it seems that I forgot